content/release-notes/1.19.6.md

```diff
@@ -0,0 +1,10 @@
+---
+date: 2020-10-02
+linktitle: "1.19.6"
+title: "1.19.6"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<fixes>}}
+* Added data migrations to address a bug that caused KOTS Admin console to not function due to missing data after upgrading from an older version.
```

Comment:
I was tempted to suggest fewer words, like:
Added data migrations to address missing data after upgrading from an older version.

But I think the more verbose version is fine.